### PR TITLE
Peterkobylarz

### PR DESCRIPTION
**Pull request by peterkobylarz**
pull request for updated readme
#27

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,3 +6,4 @@ Contributors:
 | Name | GitHub Username | Portfolio URL |
 | ---- | --------------- | ------------- |
 | Elsa Culler | eculler | https://eculler.github.io/ |
+| Peter Kobylarz | peterkobylarz | https://peterkobylarz.github.io/
```
Comment (cmq879) [inline README.md:9]: Can you add a pipe | to the end of line 9?

Comment (peterkobylarz) [reply README.md]: I've added the pipe and re-committed the change. Thanks!

**Review by misterskye — COMMENTED** [review]
Looks great!

**Review by eggvoice — APPROVED** [review]
lgtm!

**Review by nquarder — APPROVED** [review]
looks good @peterkobylarz ! 😄

**Pull request merged**